community-packages/package-list.json

```
@@ -227,6 +227,10 @@
     {
       "repoSlug": "pulumiverse/pulumi-matchbox",
       "schemaFile": "provider/cmd/pulumi-resource-matchbox/schema.json"
+    },
+    {
+      "repoSlug": "lbrlabs/pulumi-lbrlabs-eks",
+      "schemaFile": "provider/cmd/pulumi-resource-lbrlabs-eks/schema.json"
```

I don't see a schema.json in this location in the repo.  I'm not an expert on how this works, but does this work?

---

Yeah, I believe this needs to be this:

```suggestion
      "schemaFile": "schema.yaml"
```